settings/sjs.yaml

```diff
@@ -1,10 +0,0 @@
-# Please copy this file to sjs.yaml and change the connection string to
-# point to your redis instance, or just use the below default local redis connection
-# string for testing. (You still need a local redis server installed and running for this to work!)
-
-redis:
-  host: '10.201.0.11'
-  port: 6379
-  db: 2
-queue: 'ipmof'
-max_seconds_per_job: 1800
```

Comment:
this should be squashed with the last one. 

Note the theme here where I comment on a commit, then you fix it in the next one? That's why we squash things down.